csharp/test/Apache.Arrow.Tests/Date32ArrayTests.cs

```diff
@@ -20,28 +20,46 @@ namespace Apache.Arrow.Tests
 {
     public class Date32ArrayTests
     {
-        public class Set
+        public class GetDate
         {
             [Fact]
-            public void SetAndGet()
+            public void SetAndGetNull()
             {
-                var now = DateTimeOffset.UtcNow;
+                // Arrange
+                var array = new Date32Array.Builder()
+                    .AppendNull()
+                    .Build();
+
+                // Act
+                var actual = array.GetDateTime(0);
```

Comment:
Can we also get tests for `GetDateTimeOffset`?

Reply:
Added.